mysql/tests/conftest.py

```diff
@@ -143,6 +144,15 @@ def _wait_for_it_script():
     return os.path.abspath(script)
 
 
+def _mysql_conf_path():
+    filename = {
+        'mysql': 'mysql.conf',
+        'mariadb': 'mariadb.conf',
+    }[MYSQL_FLAVOR]
```

Comment:
While this pattern is helpful if the flavors expand, other parts of this file seem to just use if/then clauses which might be more clear.  e.g line 90

Reply:
Sure. I think the if/else style could use some `assert`s though, to make sure we always return a value I'll use that pattern here, but we can refactor the rest later. :-)